spec/fixtures/files/cohort_csv_data.csv

```diff
@@ -0,0 +1,2 @@
+start-year,registration-start-date,academic-year-start-date,npq-registration-start-date
```

Comment:
Is it worth putting these under a dedicated subdirectory, such as `files/importers`?